packages/react-devtools-shared/src/devtools/store.js

```diff
@@ -1444,6 +1444,12 @@ export default class Store extends EventEmitter<{
     copy(text);
   };
 
+  getAllElements = (): Array<{id: number, displayName: string | null}> => {
```

Comment:
kinda clueless about the flow error!, things are working fine, need some help here